src/components/Welcomes/Banner.tsx

```diff
@@ -27,46 +26,44 @@ interface BannerUIProps
               onChange(nextValue: string): void
           }
 }
-const useStyles = makeStyles((theme: Theme) => ({
-    root: {
-        border: `1px solid ${theme.palette.divider}`,
-        borderRadius: 4,
-        marginBottom: 10,
-    },
-    title: {
-        color: theme.palette.text.primary,
-        paddingBottom: 0,
-    },
-    wrapper: {
-        margin: theme.spacing(1),
-        display: 'flex',
-        alignItems: 'center',
-    },
-    maskicon: {
-        width: 56,
-        height: 56,
-        margin: theme.spacing(0, 1),
-    },
-}))
+const useStyles = makeStyles((theme: Theme) => {
+    const network = getActivatedUI()?.networkIdentifier
+    return {
+        root: {
+            border: `1px solid ${theme.palette.divider}`,
+            borderRadius: 4,
+            marginBottom: 10,
+            ...(network === 'twitter.com'
```

Comment:
👀👀👀

Reply:
It's cleaner than using an external theme defined in other places.